unicode-data/exe/Parser/Text.hs

```diff
@@ -16,23 +16,30 @@
 -- https://github.com/llelf/prose (Antonio Nikishaev) and heavily modified by
 -- Harendra Kumar.
 --
-module Parser.Text (genModules) where
+module Parser.Text
+    ( genCoreModules
+    , genNamesModules
+    ) where
```

Comment:
You have made significant changes to this file, you should add yourself in the copyright section above.